### PR TITLE
update gap defaults

### PR DESCRIPTION
**Pull request by Fercas123**
After a conversation with Darrin and Ben, we are using 3x as a default for every density but TD where the default will be 2x

### CODE REVIEW

File: packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx [hunk 1]
```diff
@@ -81,11 +81,9 @@ describe("GIVEN a Grid", () => {
...
         );
 
-        cy.get(".uitkGridLayout").should(
-          "have.css",
-          "grid-template-columns",
-          "72px 72px 72px 72px 72px 72px 72px 72px 72px 72px 72px 72px"
-        );
+        cy.get(".uitkGridLayout")
+          .invoke("css", "grid-template-columns")
+          .should("match", /^(\d*\.?\d*px *){12}$/);
```
Comment (JoanaMMoreira) [inline packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx:86]: 🎉

**Review by JoanaMMoreira — APPROVED** [review]

File: packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx (hunk 1 above)
Comment (origami-z) [reply packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx:86]: This may cause a little confusing for others when first seen

**Review by origami-z — APPROVED** [review]

File: packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx (hunk 1 above)
Comment (Fercas123) [reply packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx:86]: agree @origami-z, I tried extracting it to a testColumns() where you could set the number of them but cypress was timing out whenever i tried extracting it :(

Comment (Fercas123) [reply packages/lab/src/__tests__/__e2e__/layout/Grid.cy.tsx]: @origami-z I wasn't scaping  `\`s 🤦‍♀️  pushed a change to clean it :)

File: packages/lab/src/layout/FlexLayout/FlexLayout.css [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+.uitk-density-medium {
+  --uitkFlexLayout-gap-density-multiplier: 3;
+}
+.uitk-density-high {
+  --uitkFlexLayout-gap-density-multiplier: 3;
+}
+
 /* VARIABLES */
 .uitkFlexLayout {
   --uitkFlexLayout-gap-multiplier: 1;
```
Comment (libertymayc) [inline packages/lab/src/layout/FlexLayout/FlexLayout.css:16]: we tend to use this --uitkFlexLayout-gap-multiplier as a CSS API, and our own local variables as --flex-layout-gap-multiplier (see Button for example)

so you'll have something like...
.uitk-density-high {
--flex-layout-gap-density-multiplier: 3;
}
etc..
and
.uitkFlexLayout {
--flex-layout-gap-multiplier: var(--uitkFlexLayout-gap-multiplier, 1);
--flex-layout-gap: calc(var(--uitk-size-unit) * var(--flex-layout-gap-multiplier) * var(--uitkFlexLayout-gap-density-multiplier, --flex-layout-gap-density-multiplier));
}

Comment (Fercas123) [reply packages/lab/src/layout/FlexLayout/FlexLayout.css:16]: makes sense, I'll update those

**Review by origami-z — APPROVED** [review]

File: packages/lab/src/layout/GridLayout/GridLayout.css [hunk 3]
```diff
@@ -1,5 +1,18 @@
+.uitk-density-touch {
+  --uitkGridLayout-gap-density-multiplier: 2;
```
Comment (libertymayc) [inline packages/lab/src/layout/GridLayout/GridLayout.css:2]: same for these ones as flex :)

**Review by JoanaMMoreira — APPROVED** [review]

**Pull request merged**